frontend/src/component/Sidebar/WorkspaceListItem.jsx

```diff
@@ -75,7 +75,7 @@ class WorkspaceListItem extends React.Component {
 
     return (
       <li
-        id={props.workspaceId}
+        id={props.id}
```

Comment:
> > Automated tests SEEM TRICKY TO DO, if you have any idea, I'd be glad to hear from you
> 
> The tests `functionnal_tests/cypress/integration/workspace/scroll_to_active_content.js` aren't enough for the scroll?
> And an unit test for `spaceItemId` is simple to do too.

Actually, `integration/workspace/create_workspace_spec.js` is the right test. It would be nice to test with plugins activated but we don't have this enabled.

I don't see the point of testing `spaceItemId` so I'd rather not have a test for this if you agree (otherwise it's simple to do indeed).